app/panel/components/Settings/TrustAndRestrict.jsx

```diff
@@ -182,8 +180,12 @@ class TrustAndRestrict extends React.Component {
 	* @return {ReactComponent}   ReactComponent instance
 	*/
 	render() {
-		const trusted_sites = this.props.site_whitelist;
-		const restricted_sites = this.props.site_blacklist;
+		const { actions, site_whitelist, site_blacklist } = this.props;
+		const {
+			menu, trustedValue, currentWarning, restrictedValue
+		} = this.state;
+		const trusted_sites = site_whitelist;
+		const restricted_sites = site_blacklist;
```

Comment:
After destructuring, is there any reason not to use `site_whitelist` and `site_blacklist` directly?

Reply:
I can't see why not.  Updated on feature/cleanup-cleanup.

Reply:
Thanks, lgtm in `feature/cleanup-cleanup`!